modules/affile/file-reader.c

```diff
@@ -235,7 +235,6 @@ _reopen_on_notify(LogPipe *s, gboolean recover_state)
   FileReader *self = (FileReader *) s;
 
   _deinit_sd_logreader(self);
-  log_reader_enable_bookmark_saving(self->reader);
```

Comment:
Is there a reason to keep this change around, or it could be squashed into the commit that introduces it? (And maybe delete the related api exposing functions together?)